CentOS-6-32/Dockerfile

```diff
@@ -22,6 +21,12 @@ RUN yum -y update; yum -y install \
                    tar \
                    zlib-devel.i686
 RUN yum clean all
+RUN curl -fSL "https://cmake.org/files/v3.1/cmake-3.1.3-Linux-i386.sh" -o /usr/cmake.sh
+RUN cd /usr \
+ && chmod +x ./cmake.sh
+
+RUN cd /usr \
+&& ./cmake.sh --skip-license
```

Comment:
That does not work because the script `cmake.sh` tries to reopen itself with `tail +199 | gunzip | tar xf`, and that cannot work if the file is only kept in memory in a pipeline buffer.

Then, use only one `RUN`, to save space:
```dockerfile
RUN curl -fSL "https://cmake.org/files/v3.1/cmake-3.1.3-Linux-i386.sh" -o cmake.sh && bash cmake.sh --skip-license && rm cmake.sh
```